assets/scss/core/_reset.scss

```diff
@@ -11,12 +11,12 @@ RESET DOPPLER STYLE GUIDES
  */
 ::-moz-selection {
   background: #fbb224;
-  color: #FFF;
+  color: #fff;
```

Comment:
Why the colors in lower case? cannot it be configured somehow to allow this?

Reply:
Is the same, if prettier suggest lowercase, why not? It is an opinionated code formatter.